index.html

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>ToDoApp</title>
-</head>
-<body>
-    
-
+  </head>
+  <body>
+    <h1>ToDoApp</h1>
+    <h5 id="message" hidden></h5>
+    <h3>Add Tasks</h3>
```

Comment:
```suggestion
    <p>Add Tasks</p>
```
Let's use a p tag instead of h as this is not a header.